packages/flutter_tools/lib/src/commands/build_aot.dart

```diff
@@ -256,44 +253,27 @@ Future<String> _buildAotSnapshot(
     printError('Missing input files: $missingInputs');
     return null;
   }
-  if (!processManager.canRun(genSnapshot)) {
-    printError('Cannot locate the genSnapshot executable');
-    return null;
-  }
 
-  final List<String> genSnapshotCmd = <String>[];
-  // iOS gen_snapshot is a multi-arch binary. Running as an i386 binary will
-  // generate armv7 code. Running as an x86_64 binary will generate arm64
-  // code. /usr/bin/arch can be used to run binaries with the specified
-  // architecture.
-  //
-  // TODO(cbracken): update the GenSnapshot class to handle AOT builds.
-  if (platform == TargetPlatform.ios)
-    genSnapshotCmd.addAll(<String>['arch', '-x86_64']);
-  genSnapshotCmd.addAll(<String>[
-    genSnapshot,
-    '--await_is_keyword',
+  final List<String> genSnapshotArgs = <String>[];
+  genSnapshotArgs.addAll(<String>[
```

Comment:
Just initialize the list with these args

Reply:
Oops! good catch!